grafana-init/grafana.py

```diff
@@ -58,19 +58,22 @@ def f_retry(*args, **kwargs):
         return f_retry
     return decorator
 
-
```

Comment:
Don't remove this empty line, flake8 (linter) will complain that before every `def` (or `@retry` before `def`) we need two empty lines.

Reply:
Sorry for late update.
Done.